courses/1-ramda-basics/1-intro.js

```diff
@@ -95,7 +95,8 @@ describe("Course 1 - Ramda Basics :: Lesson 1 - Intro", () => {
         const solution = R.map(multiplyBy2, numbers);
 
         function multiplyBy2 (x) {
-          throw new Error("This function is incomplete");
+        return x * 2
+        throw new Error("This function is incomplete");
```

Comment:
I probably should have made the instructions more clear but the error line should be deleted. While it will not run due to the early return above it, leaving in statements like that can make it difficult to understand the intent of the function.

Reply:
Another issue is that the body of the function should be indented. It's not a hard requirement but it's a common JS practice to be mindful of. In an asynchronous language like JavaScript the hierarchy of blocks, anything wrapped in a multline pair of `{` and `}`, can make a big difference when code should be executed.

Installing a JS linter in Atom will help give you immediate feedback to follow the general JS formatting conventions and in some cases catch issues before you run it.